kvmagent/kvmagent/plugins/vm_plugin.py

```
@@ -10296,8 +10296,8 @@ def detach_pci_device_from_vm(cmd, xml_path, vm_domain):
             if not find_pci_device(cmd.vmUuid, cmd.pciDeviceAddress):
                 return
 
-            # Pre-detach if GPU
-            if pci.is_gpu(cmd.type):
+            # Pre-detach if GPU（只基于实际工具返回）
```

Ruff (0.14.13)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)